datafusion/functions/src/regex/regexpcount.rs

```diff
@@ -655,11 +657,12 @@ mod tests {
             let v_sv = ScalarValue::Utf8(Some(v.to_string()));
             let regex_sv = ScalarValue::Utf8(Some(regex.to_string()));
             let expected = expected.get(pos).cloned();
-            #[allow(deprecated)] // TODO: migrate to invoke_with_args
-            let re = RegexpCountFunc::new().invoke_batch(
-                &[ColumnarValue::Scalar(v_sv), ColumnarValue::Scalar(regex_sv)],
-                1,
-            );
+            #[allow(deprecated)]
```

Comment:
I think it is out of date. It's for `inovke_batch` but we have migrated to `inovek_with_args`. We can remove it.

Reply:
There are other similar parts. We can remove all of them.